release-notes/swan-lake-betarc1-release-note.md

```diff
@@ -55,12 +55,29 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta RC1](https://git
 
 ### Standard Library Updates
 
-#### New Features
-
 #### Improvements
 
+##### GraphQL Package
+
+ - Allow enum as an input parameter.
+
 #### Bug Fixes
 
+##### GraphQL Package
```

Comment:
```suggestion
##### GraphQL Package
```
We do not mention bug fixes specifically in the current format.

Reply:
Removed.